install/hyprlandia.sh

```diff
@@ -3,5 +3,20 @@ yay -S --noconfirm --needed \
   wofi waybar mako swaybg \
   xdg-desktop-portal-hyprland xdg-desktop-portal-gtk
 
+# Checks if nvidia gpu exist
+gpu_info=$(lspci | grep -i 'nvidia')
+if [ -n "$gpu_info" ]; then
+    # Ask user if they want to install NVIDIA drivers
+    if [ -t 0 ]; then
+        read -p "NVIDIA GPU detected. Do you want to install NVIDIA drivers? (y/N) " install_nvidia
```

Comment:
Don't think we should ever ask. If we detect nvidia GPU, we install nvidia drivers.